lmms_eval/tasks/docqa_hu_syn/utils.py

```diff
@@ -0,0 +1,43 @@
+import json
+import os
+import logging
+
+from lmms_eval.tasks._task_utils.file_utils import generate_submission_file
+
+logger = logging.getLogger("lmms-eval")
+
+
+def docvqa_doc_to_visual(doc):
+    return [doc["image"].convert("RGB")]
+
+
+def docvqa_doc_to_text(doc, model_specific_prompt_kwargs):
+    question = doc["conversations"][0]["value"]
+    pre_prompt = model_specific_prompt_kwargs["pre_prompt"]
+    post_prompt = model_specific_prompt_kwargs["post_prompt"]
+    return f"{pre_prompt}{question}{post_prompt}"
```

Comment:
Nidhi's prompts have a double-escaped newline `\\n` separating the image and prompt, while Fenglu's have no space separating the image and prompt. Can we normalize those differences?